pkg/metacni/metacni.go

```diff
@@ -367,6 +367,12 @@ func createDelegatedInterface(syncher *syncher.Syncher, danmClient danmclientset
     syncher.PushResult(netInfo.ObjectMeta.Name, errors.New("DanmEp object could not be created due to error:" + err.Error()), nil)
     return
   }
+  err = danmep.PutDanmEp(danmClient, ep)
```

Comment:
yeah I was thinking about how to approach this, and yes this is one way to do it :)

my issue with this approach is that we add an extra REST API call for ALL successful transactions, because SOME of them might fail. albeit not much, but it is extra latency

instead, I was thinking about leaving the existing DanmEp creation as-is. Instead I would simply add a PutDanmEp call into the error branch DelegateInterfaceSetup.
Same effect, but without the extra leg for the successful calls - which are the majority